medusa/server/api/v2/internal.py

```diff
@@ -53,24 +51,27 @@ def get(self, resource, path_param=None):
         data = resource_function()
         return self._ok(data=data)
 
+    # existingSeries
     def resource_existing_series(self):
         """Generate existing series folders data for adding existing shows."""
-        root_dirs = json_decode(self.get_argument('root-dir', '[]'))
-
-        if not root_dirs:
-            if app.ROOT_DIRS:
-                root_dirs = app.ROOT_DIRS[1:]
-            else:
-                self._not_found('No configured root dirs')
-
-        # Put the default root-dir first
-        try:
-            default_index = int(app.ROOT_DIRS[0])
-            default_root_dir = root_dirs[default_index]
-            root_dirs.remove(default_root_dir)
-            root_dirs.insert(0, default_root_dir)
-        except (IndexError, ValueError):
-            pass
+        root_dirs_indices = self.get_argument('rootDirs', '')
+
+        if root_dirs_indices:
+            root_dirs_indices = set(root_dirs_indices.split(','))
```

Comment:
Not a biggie, but I think this can go just after line 66? One `if` less 🎉 
The rest of the python code LGTM.

Reply:
Yes, looks like it to me too. Good catch! Thanks.
I will fix it as soon as I can.